### DISCUSSION TITLE
Custom layouts in non-docs directories

### DISCUSSION THREAD

**Discussion by reavpC6K** [imfing/hextra#416] [Q&A]
I edit the index without a problem, but how do you create layouts for non docs folders?  
```  
.
├── exampleSite/
│   └── content/
│       ├── index.md (layout:index) OK
│       ├── foo/
│       │   ├── _index.md (layout: foo) also tried cascade, 404
│       │   ├── _index.fr.md (layout: foo) 404
│       │   └── ...
│       └── bar/
│           ├── _index.md (layout: bar) 404
│           ├── _index.fr.md (layout: bar) 404
│           └── ...
└── layouts/
    ├── foo.html
    ├── bar.html
    └── index.html
```

**Comment by imfing** [comment]
please check https://gohugo.io/templates/lookup-order/#section-templates 

I think you need to put the layouts in different directories to make hugo pick it up

**Comment by reavpC6K** [comment]
It didn't work but I solved it using the `wide` layout and shortcodes.  

```
---
title: Apples
layout: wide
toc: false
sidebar:
  exclude: true
---
{{< apples >}}
```
```
---
title: Pommes
layout: wide
toc: false
sidebar:
  exclude: true
---
{{< apples >}}
```
For i18n within shortcodes:  
```
<p>{{ i18n "KeyName" | safeHTML }}</p>  
```  

-Conflicting styles should be #scoped and !important in assets/css/custom.css  
-I had to rename index.md to _index.md, and back, to get around 404's  
-As a result I deployed public without build as that would create first-time 404's.  
-Sitemaps were pretty broken even with `--baseURL` so I used [mcmilk/sitemap-generator](https://github.com/mcmilk/sitemap-generator)  
  
This template is fantastic for docs but nothing else. Will showcase later.
